src/bootstrap/format.rs

```diff
@@ -1,7 +1,196 @@
 //! Runs rustfmt on the repository.
 
-use crate::Build;
+use {
+    crate::{util, Build, Crate},
+    glob::glob,
+    serde::Deserialize,
+    std::{
+        collections::BTreeSet,
+        fs,
+        path::{Path, PathBuf},
+        process::Command,
+    },
+};
```

Comment:
If rustfmt does this or you've done this we should alter settings to go with `use ...` on a single line.

Reply:
Oh this is just the style that's common at work, habitual. I can change it to single-line imports.

Reply:
(The reason we prefer the single-line style is because it means `rg use.*<Ident>` works with it, and the multiline style is much harder to search for).